packages/core/tests/unit/chart-builders/dimension-test.ts

```diff
@@ -0,0 +1,339 @@
+/* eslint-disable @typescript-eslint/camelcase */
```

Comment:
What is this guy for?

Reply:
This is to ignore using our cids like `cid_totalPageViews` as keys in an object
https://github.com/yahoo/navi/blob/05267c315958279ae2025456f61401623814ced9/packages/core/tests/unit/chart-builders/dimension-test.ts#L88